ui/ui-components/components/property/profilePreview.tsx

```diff
@@ -1,10 +1,17 @@
-import { useState, useEffect } from "react";
-import { useApi } from "../../hooks/useApi";
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
 import { Card, ListGroup } from "react-bootstrap";
+import { useApi } from "../../hooks/useApi";
+import { Actions } from "../../utils/apiData";
 import Loader from "../loader";
 import ProfileImageFromEmail from "../visualizations/profileImageFromEmail";
-import { useRouter } from "next/router";
-import { Actions } from "../../utils/apiData";
+
+const calculatePreviewDate = (date: any) => {
+  if (!date) {
+    return new Date().toLocaleString();
```

Comment:
Unsure what we want the default behavior to be. Assuming current time for now.

Reply:
We might want a `""` here - This component is trying to render the Properties of a Profile (e.g.: "Most Recent Purchase Timestamp").  If they don't have a purchase we want to show null, "", or "N/A" and not imply that they've made a purchase

Reply:
https://www.pivotaltracker.com/story/show/179394479
I left a comment asking what to do. I'm good with any of those.

Reply:
go ahead and return null here or whatever the other ones do. I imagine it's null.
It might be the most correct to check for `date === null || date === undefined` here as I suppose a date of `0` is supported.